Source/CarthageKit/Xcode.swift

```diff
@@ -891,15 +896,16 @@ public func buildDependencyProject(dependency: ProjectIdentifier, rootDirectoryU
 	return (buildOutput, copyProducts)
 }
 
-/// Augments a no shared schemes error description to include a suggestion
-/// to file a GitHub issue in GitHub repositories.
+/// Augments an error localized description to include a suggestion to file a
+/// GitHub issue for repositories with potentially no shared schemes.
 ///
-/// Returns a no shared schemes error with an issue filing suggestion.
+/// Returns an error with an issue filing suggestion.
 private func addIssueFilingSuggestionToError(error: NSError, repo: GitHubRepository) -> NSError {
-	let msg = error.localizedDescription + "\n\nIf you believe this to be an error, please file an issue with the maintainers at \(repo.newIssueURL.absoluteString!)"
-	return CarthageErrorCode.NoSharedSchemes.error([
-		    NSLocalizedDescriptionKey: msg
-		])
+	var userInfo = error.userInfo!
```

Comment:
Here I’m assuming that this is a Carthage error which a localised description. Is it better to fail when this is not the case or to work around in case there is no localised description?